types/globals.d.ts

```diff
@@ -0,0 +1,5 @@
+interface Window {
```

Comment:
Not sure if this file is still required after refactoring everything to use `loadConfig` instead, but if it helps the compiler you can leave it in.